lib/binaries/config_source.ts

```diff
@@ -157,7 +157,11 @@ export abstract class GithubApiConfigSource extends JsonConfigSource {
           });
 
         } else {
-          reject(new Error('response status code is not 200'));
+          if (response.statusCode == 403 && response.headers['x-ratelimit-remaining'] == 0) {
```

Comment:
Please fix this conditional. This should be combined to the conditional above as:
```
} else if (response.statusCode == 403 && response.headers['x-ratelimit-remaining'] == 0) {
```

Similar nit for the `else` statement below.